### PR TITLE
Dialog To Show Store Ratings in Compare Stores Flow

### PR DESCRIPTION
**Pull request by workla**
- Dialog pops up and shows store details, average rating and recyclerview of comments
- Response created for when API is up and ready and all locations consume dummy response at the moment
- Back button in dialog dismisses dialog to return to compare store detail fragment

### CODE REVIEW

File: app/src/main/java/com/example/cssafrontend/networkLogic/responses/ProfileDataResponse.kt [hunk 1]
```diff
@@ -15,4 +14,13 @@ class GroceryList {
     var listId:  Int = 0
     @SerializedName("itemCount")
     var itemCount:  Int = 0
+}
+
+class User {
```
Comment (workla) [inline app/src/main/java/com/example/cssafrontend/networkLogic/responses/ProfileDataResponse.kt:19]: Another response update prepping for API integration

File: app/src/main/java/com/example/cssafrontend/networkLogic/responses/StoreReviewsResponse.kt [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+class Review {
+    @SerializedName("user")
+    var user: User = User()
+    @SerializedName("review")
+    var review: ReviewDetail = ReviewDetail()
+}
+
+class ReviewDetail {
```
Comment (workla) [inline app/src/main/java/com/example/cssafrontend/networkLogic/responses/StoreReviewsResponse.kt:22]: Right now, I only use text out of this.....just mimicing what I see in API doc.  If any others need to disappear, very easy to accomplish

Comment (rsandfield) [reply app/src/main/java/com/example/cssafrontend/networkLogic/responses/StoreReviewsResponse.kt:22]: You're not using the rating anywhere? I see you have the user in the wrapping Review class.

**Review by rsandfield — APPROVED** [review]
Have a question on your comment but it's not blocking.

**Comment by workla** [comment]
Ya I don't currently yse rating anywhere....but thought I would end up using in the button in profile that pops up the dialog explaining reputation.  Something like "you are a ### from your contributions....up # of contribs and you get to up that score, which translates to a better reputation.  BUt haven't gotten there yet.  All our wireframes only showed reputation so far

**Pull request merged**